recipe/patch_yaml/fmt.yaml

```diff
@@ -0,0 +1,18 @@
+# Extend the repodata patch made on `win-64` to `linux-64` as well.
+#
+# See:
+#  - https://github.com/conda-forge/fmt-feedstock/issues/61
+#  - https://github.com/conda-forge/admin-requests/pull/1433
+#  - https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/984/
+#  - https://github.com/conda-forge/admin-requests/pull/1468#issuecomment-2769312616
+if:
+  timestamp_lt: 1743030622335
+  has_depends: fmt >=11.0*
+  subdir_in: [win-64, linux-64]
```

Comment:
Going forward, please properly rebase your patches on top of the HEAD of main rather than recreating files that already exist (I didn't notice this when looking at the diff).